server/src/utils/fight/generateFight.ts

```diff
@@ -359,13 +371,15 @@ const generateFight = async ({
           count: 1,
         })),
       });
-
+      
+      // Filter out Cerberus from potential bosses most of the time
+      const potentialBosses = bosses.filter((boss) => boss.name != BossName.Cerberus || Math.random() * 4 < 1)
```

Comment:
While this works, it makes the code hard to maintain if we want to change tosses later.
Instead, add a `toss` value to the defined boss list, set cerberus to 0, and use a weighted random instead of a random item.
This will effectively prevent cerberus from being rolled, and allow for later rarer boss if needed